geometry/geometry_config.py

```diff
@@ -131,8 +131,8 @@
      c.chambers.Rmin = 245.*u.cm
      c.chambers.Rmax = 250.*u.cm
      # positions and lenghts of vacuum tube segments
-     zset=z4-4666.*u.cm-magnetIncrease-extraVesselLength
-     c.Chamber1 = AttrDict(z=zset)
+     zset = z4 - 4810.75*u.cm - magnetIncrease - extraVesselLength
+     c.Chamber1 = AttrDict(z = zset)
```

Comment:
```suggestion
     c.Chamber1 = AttrDict(z=zset)
```

Keyword arguments unfortunately are an exception to the rule of a space around operators.
Imagine this comment repeated for the other occurrences.